apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -431,6 +445,9 @@ export const PageTemplatesPlugin = compose(
 				} );
 			},
 			insertTemplate: ( title, blocks ) => {
+				// Add filter to let the tracking library know we are inserting a template.
+				addFilter( INSERTING_HOOK_NAME, INSERTING_HOOK_NAMESPACE, stubTrue );
```

Comment:
With `stubTrue` being the default value?

Reply:
yeah, we need it to always return true and this is a convenience method for just that. we remove the filter after insertion so it will be back to `false`